### PR TITLE
Update molpay.rb & helper_test.rb

### PR DESCRIPTION
**Pull request by frankiecmk**
added notify_url and update unit test.

### CODE REVIEW

**Comment by frankiecmk** [comment]
@andrewpaliga helper_test.rb updated.

File: lib/offsite_payments/integrations/molpay.rb [hunk 1]
```diff
@@ -12,6 +12,10 @@ def self.return(query_string, options={})
         Return.new(query_string, options)
       end
 
+      def self.notification(post, options = {})
```
Comment (andrewpaliga) [inline lib/offsite_payments/integrations/molpay.rb]: Why do you need this?

Comment (frankiecmk) [inline lib/offsite_payments/integrations/molpay.rb]: duplicated, will remove and create a pull request.

**Comment by frankiecmk** [comment]
@andrewpaliga molpay.rb updated.

**Comment by frankiecmk** [comment]
@andrewpaliga any update on this?

**Comment by andrewpaliga** [comment]
:+1:

**Pull request merged**